docker-entrypoint.sh

```diff
@@ -80,7 +80,16 @@ if expr "$*" : '^java .*/start\.jar.*$' >/dev/null ; then
 		set -- $(cat /jetty-start)
 	else
 		# Do a jetty dry run to set the final command
-		set -- $("$@" --dry-run | sed 's/\\$//' )
+		"$@" --dry-run > /jetty-start
+		EXIT=$?
+		if [ $(egrep -v '\\$' /jetty-start | wc -l ) -gt 1 ] ; then
```

Comment:
The annoying thing is that jetty-9.2 --dry-run doesn't produce a single line. Instead it produces multiple lines split with a final `\` character.  So this egrep to wc is counting the number of lines that do not terminate with `\` (and specifically it is not anchered with `^`).